ddp-workspace/projects/ddp-dsm-ui/src/app/participant-page/participant-page.component.ts

```diff
@@ -828,13 +828,19 @@ export class ParticipantPageComponent implements OnInit, OnDestroy, AfterViewChe
                 oncHis.faxSentBy = this.role.userID();
                 this.oncHistoryValueChanged(oncHis.faxSent, 'faxSent', oncHis);
               } else if (oncHis.faxSent2 == null) {
-                oncHis.faxSent2 = Utils.getFormattedDate(date);
-                oncHis.faxSent2By = this.role.userID();
-                this.oncHistoryValueChanged(oncHis.faxSent2, 'faxSent2', oncHis);
+                //If current date is not already on FaxSent1
+                if(oncHis.faxSent != Utils.getFormattedDate(date)) {
+                  oncHis.faxSent2 = Utils.getFormattedDate(date);
```

Comment:
it would be nicer if you pull `Utils.getFormattedDate(date)` into it's own parameter and then compare the faxSents to that parameter instead of always calling the same method over and over

Reply:
@SimoneMaiwald I refactored the code, let me know if that is what you had in mind